<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="setting_button">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>15</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Settings&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;
border-color: rgba(50, 50, 150, 100);
background-color: none;
color: rgb(200, 200, 255);
font: 75 15pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string>⚙</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setting_button_3">
    <property name="geometry">
     <rect>
      <x>679</x>
      <y>8</y>
      <width>215</width>
      <height>59</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiLight</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select Weather Provider&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;
border: 1px solid white;
border-color: rgb(66, 79, 122);
background-color: none;
color: rgb(200, 200, 255);
padding:10px;</string>
    </property>
    <property name="text">
     <string>AccuWeather</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_sunrise">
    <property name="geometry">
     <rect>
      <x>389</x>
      <y>379</y>
      <width>111</width>
      <height>37</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiLight</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Sunrise Time</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;6:00&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_pressure">
    <property name="geometry">
     <rect>
      <x>263</x>
      <y>243</y>
      <width>91</width>
      <height>46</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiLight</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Current Pressure</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>228</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_humidity">
    <property name="geometry">
     <rect>
      <x>263</x>
      <y>161</y>
      <width>91</width>
      <height>46</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiLight</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Current Humidity</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>007%</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_temp">
    <property name="geometry">
     <rect>
      <x>328</x>
      <y>220</y>
      <width>278</width>
      <height>135</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiLight</family>
      <pointsize>72</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>2.2°</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="l_sunset">
    <property name="geometry">
     <rect>
      <x>546</x>
      <y>376</y>
      <width>111</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiLight</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Sunset Time</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>19:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>315</x>
      <y>140</y>
      <width>301</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiLight</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Current Time</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>13:37</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="l_wind_speed">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>262</x>
      <y>323</y>
      <width>111</width>
      <height>46</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiLight</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Current Wind Speed</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>1488</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>330</y>
      <width>32</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/wind.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>250</y>
      <width>32</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/barometer.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>32</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/drop.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>170</y>
      <width>32</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/ultraviolet.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="l_uv_index">
    <property name="geometry">
     <rect>
      <x>576</x>
      <y>164</y>
      <width>81</width>
      <height>46</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiLight</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Current UV Index</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>1488</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>246</y>
      <width>32</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/sunny.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="l_day_temp">
    <property name="geometry">
     <rect>
      <x>576</x>
      <y>240</y>
      <width>81</width>
      <height>46</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiLight</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Temperature at Day</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>30°</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>668</x>
      <y>326</y>
      <width>32</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/night-mode.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="l_night_temp">
    <property name="geometry">
     <rect>
      <x>584</x>
      <y>320</y>
      <width>71</width>
      <height>46</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiLight</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Temperature at Night</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>10°</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>380</y>
      <width>32</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/sunrise.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>380</y>
      <width>32</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/sunset.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>900</width>
      <height>600</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/background.jpg</pixmap>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="cityNameField">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>10</y>
      <width>215</width>
      <height>59</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift SemiLight</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(200, 200, 255);
padding: 10px;
border: 1px solid white;
border-color: rgb(66, 79, 122);
border-radius: 15px;</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="lineWrapMode">
     <enum>QPlainTextEdit::NoWrap</enum>
    </property>
    <property name="plainText">
     <string>Иркутск</string>
    </property>
   </widget>
   <zorder>label_14</zorder>
   <zorder>setting_button</zorder>
   <zorder>setting_button_3</zorder>
   <zorder>l_sunrise</zorder>
   <zorder>l_pressure</zorder>
   <zorder>l_humidity</zorder>
   <zorder>l_sunset</zorder>
   <zorder>label_3</zorder>
   <zorder>l_wind_speed</zorder>
   <zorder>l_temp</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>l_uv_index</zorder>
   <zorder>label_8</zorder>
   <zorder>l_day_temp</zorder>
   <zorder>label_9</zorder>
   <zorder>l_night_temp</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
   <zorder>cityNameField</zorder>
  </widget>
 </widget>
 <resources>
  <include location="bebr.qrc"/>
 </resources>
 <connections/>
</ui>
